bleak/backends/bluezdbus/client.py

```diff
@@ -570,13 +572,56 @@ def is_connected(self) -> bool:
             False if self._bus is None else self._properties.get("Connected", False)
         )
 
+    async def _acquire_mtu(self) -> None:
+        """Acquires the MTU for this device by calling the "AcquireWrite" or
+        "AcquireNotify" method of the first characteristic that has such a method.
+
+        This method only needs to be called once, after connecting to the device
+        but before accessing the ``mtu_size`` property.
+        """
+        # This will try to get the "best" characteristic for getting the MTU.
```

Comment:
I was seeing some issues when the characteristic was encrypted and the device was not bonded. Usually, an unsufficient-auth error causes a disconnect. Have you tried that? I don't think there is a workaround, but maybe worth mentioning in the doc, or example?

Reply:
I don't have any devices that use encryption so I haven't seen that. But I can add something to the doc comment.